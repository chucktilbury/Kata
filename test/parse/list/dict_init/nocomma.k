;-------------------------------------------------------------------------------
; no comma
;-------------------------------------------------------------------------------

var dict name = ['asd':123 'asd':123]