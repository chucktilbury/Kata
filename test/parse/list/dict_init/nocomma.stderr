Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/nocomma.k: 5: 33: expected a ',' or a ']' but got literal sstrg
Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/nocomma.k: 5: 34: expected module item or end of file but got :
Syntax: /home/chuck/Src/Kata/test/parse/list/dict_init/nocomma.k: 5: 34: module cannot be empty
